test/es-module/test-cjs-esm-warn.js

```diff
@@ -1,64 +1,71 @@
 'use strict';
 
-const common = require('../common');
-const fixtures = require('../common/fixtures');
-const { spawn } = require('child_process');
-const assert = require('assert');
-const path = require('path');
+const { mustCall } = require('../common');
+const fixtures = require('../common/fixtures.js');
+const assert = require('node:assert');
+const path = require('node:path');
+const { execPath } = require('node:process');
+const { describe, it } = require('node:test');
+
 
 const requiringCjsAsEsm = path.resolve(fixtures.path('/es-modules/cjs-esm.js'));
 const requiringEsm = path.resolve(fixtures.path('/es-modules/cjs-esm-esm.js'));
 const pjson = path.resolve(
   fixtures.path('/es-modules/package-type-module/package.json')
 );
 
-{
-  const required = path.resolve(
-    fixtures.path('/es-modules/package-type-module/cjs.js')
-  );
-  const basename = 'cjs.js';
-  const child = spawn(process.execPath, [requiringCjsAsEsm]);
-  let stderr = '';
-  child.stderr.setEncoding('utf8');
-  child.stderr.on('data', (data) => {
-    stderr += data;
-  });
-  child.on('close', common.mustCall((code, signal) => {
-    assert.strictEqual(code, 1);
-    assert.strictEqual(signal, null);
+(async () => {
```

Comment:
Wouldn't it be easier to refactor this to ESM to get TLA/static imports?

Reply:
I did and you said to not, so I backed it out lol

Reply:
The issue is that GitHub UI doesn't give an option to ignore the whitespace/indentation changes (*I think*?), if the only diff was adding the `await import` that would be much easier to review to keep it as CJS. As it is now, I doubt it makes much difference. Oh well, let's keep it as is, it can always be re-refactored later.

Reply:
> The issue is that GitHub UI doesn’t give an option to ignore the whitespace/indentation changes (_I think_?)

Add `?w=1` to the end of the URL. Or use one of the other methods: https://stackoverflow.com/a/37145215/223225.

Reply:
"Hide whitespace" via the GitHub UI (on the file view screen):

![image](https://user-images.githubusercontent.com/3012099/180961313-2960aa33-e1aa-4669-9087-e85ca64a8d64.png)

Damn, sorry I had added a note to my PR description about hiding whitespace changes (it reduces the diff by 36%). I dunno what happened to that note 😖

Reply:
🤦‍♂️ foiled by my own cleverness: who can spot the problem

<img width="821" alt="image" src="https://user-images.githubusercontent.com/3012099/180962173-941f48c4-04b7-4d8a-a05a-dd6e626d1142.png">

Reply:
Ah, that explains why I was never pinged about this 😄